src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header-logo > a {
+  display: block;
+  cursor: pointer;
+}
+
+.header-navigation {
+  display: flex;
+  align-items: center;
+  height: 60px;
```

Comment:
it's better not hardcode height for elements with content inside. 
Add top/bottom padding to your links and you'll get the right height